schema/auth.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$id": "https://serverlessworkflow.io/schemas/0.8/auth.json",
+  "$id": "https://serverlessworkflow.io/schemas/0.9/auth.json",
```

Comment:
Shouldn't version be updated at release time only, as referenced uri does not yet exist?

@ricardozanini @fjtirado @JBBianchi

Reply:
hmmm, I found confusing keeping the 0.8 there, but as you prefer

Reply:
Just asking. I saw that @ricardozanini requested it in my PR, too, so let's go with it.